CODEOWNERS: Make PullAssigner the default owner

sameo opened the pull request:
The PullAssigner bot [1] will pick 2 github handles from the gate
keepers list [2] and assign them for reviewing any new PR.
The bot goes uses a round robin algorithm to select handles from the
list.

What happens when the CODEOWNERS file gets populated with real
owners? Following the PR review and approval process [3], the
PullAssigner bot will either pick only one or zero reviewers from the
gate keepers list, if the CODEOWNERS file points to exactly one or more
than one actual ownwer, respectively.

[1] https://pullpanda.com/assigner
[2] https://github.com/rust-vmm/community/blob/master/GATEKEEPERS.md
[3] https://github.com/rust-vmm/community#pr-review-and-approval

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>

sameo commented:
@andreeaflorescu Could you please have a look?

andreeaflorescu commented:
@sameo it looks like this crate does not pass the tests any more. Before merging this PR we probably need to fix it.

sameo commented:
> @sameo it looks like this crate does not pass the tests any more. Before merging this PR we probably need to fix it.

Yep. Fixed and this PR is now rebased on top of the fix.
